registries.conf

```diff
@@ -0,0 +1,23 @@
+# Search registries are used when the caller of a container runtime does not fully specify the
+# container image that they want to execute.  These registries are prepended onto the front of the
+# specified container image until the named image is found at a registry.
+[registries.search]
+registries = []
+
+# Insecure Registries.  By default container runtimes use TLS when retrieving images from a registry.  If
+# registry is not setup with TLS, then the container runtime will fail to pull images from the registry.
+# if you add the registry to the list of insecure registries then the container runtime will attempt use
+# standard web protocols to pull the image.  Note insecure registries can be used for any registry, not just
+# the registries listed under search
+[registries.insecure]
+registries = []
+
+# Blocked Registries, blocks the `docker daemon` from pulling from the blocked registry.  If you specify
+# all, then the docker daemon will only be allowed to pull from registries listed above in the search
```

Comment:
What does “specify all” mean? Literally the string `"all"` instead of a real hostname?

Reply:
According to the RHEL documentation, a "*" can be used for all.  I'll update accordingly.